packages/dso-toolkit/src/components/toggletip/toggletip.models.ts

```diff
@@ -1,5 +1,5 @@
 export interface Toggletip<TemplateFnReturnType> {
-  children: TemplateFnReturnType;
+  children: TemplateFnReturnType | string;
```

Comment:
Hier heb ik 'n beetje vals gespeeld. In principe is dit niet iets wat we willen, `children` is immers een framework node. Maar omdat die Angular Storybook ons nu dusdanig in de weg zit is dit ff makkelijker.